notebook/templates/notebook.html

```diff
@@ -89,6 +89,9 @@
                         <li id="copy_notebook"
                             title="{% trans %}Open a copy of this notebook's contents and start a new kernel{% endtrans %}">
                             <a href="#">{% trans %}Make a Copy...{% endtrans %}</a></li>
+                        <li id="save_notebook_as"
+                            title="{% trans %}Save a copy of the notebook's contents and start a new kernel{% endtrans %}">
+                            <a href="#">{% trans %}Save as{% endtrans %}</a></li>
```

Comment:
This label should end `...` to indicate that it brings up a dialog rather than acting immediately.